docker/webui/run_openqa.sh

```diff
@@ -31,12 +31,12 @@ function webui() {
 
 function all_together_apache() {
   # run services
-  start_daemon -u geekotest /usr/share/openqa/script/openqa-scheduler &
-  start_daemon -u geekotest /usr/share/openqa/script/openqa-websockets &
-  start_daemon -u geekotest /usr/share/openqa/script/openqa-livehandler &
-  start_daemon -u geekotest /usr/share/openqa/script/openqa gru -m production run &
+  su geekotest -c /usr/share/openqa/script/openqa-scheduler-daemon &
+  su geekotest -c /usr/share/openqa/script/openqa-websockets-daemon &
+  su geekotest -c /usr/share/openqa/script/openqa-gru -m production run &
```

Comment:
Don't pass any parameters here:
```suggestion
  su geekotest -c /usr/share/openqa/script/openqa-gru &
```

Really, just do what the systemd services do (e.g. check `systemctl cat openqa-gru`). The arguments `-m production run` are already passed by `openqa-gru` so they shouldn't be repeated here.